ingestion-beam/src/test/java/com/mozilla/telemetry/util/NormalizeTest.java

```diff
@@ -0,0 +1,47 @@
+package com.mozilla.telemetry.util;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class NormalizeTest {
+
+  @Test
+  /**
+   * See a selection of existing raw channel names and their normalized versions in BigQuery:
+   *
+   * <pre>{@code
+   * SELECT channel, normalized_channel, count(*)
+   * FROM `moz-fx-data-derived-datasets.telemetry.clients_daily_v6`
+   * WHERE true
+   *   and submission_date_s3 >= DATE('2018-07-01')
+   *   and normalized_channel != 'Other'
+   * group by 1, 2
+   * order by 3 desc
+   * LIMIT 1000
+   * }</pre>
+   */
+  public void channel() {
+    assertEquals("release", Normalize.channel("release"));
+    assertEquals("esr", Normalize.channel("esr"));
+    assertEquals("beta", Normalize.channel("beta"));
+    assertEquals("aurora", Normalize.channel("aurora"));
+    assertEquals("nightly", Normalize.channel("nightly"));
+
+    assertEquals("nightly", Normalize.channel("nightly-cck-ubuntu"));
+
+    assertEquals("Other", Normalize.channel("no"));
+    assertEquals("Other", Normalize.channel("esr52"));
+    assertEquals("Other", Normalize.channel("Release"));
+    assertEquals("Other", Normalize.channel("Prefeitura Municipal de Santos"));
+    assertEquals("Other", Normalize.channel("no"));
+    assertEquals("Other", Normalize.channel(null));
+
+    assertEquals("beta", Normalize.channel("beta-cdntest"));
+    assertEquals("beta", Normalize.channel("betastop"));
```

Comment:
@trink @whd 

The existing lua implementation accepts anything prefixed with "beta" (no dash required) as "beta" channel. Do we want to preserve that behavior? Do we have documentation for the history of why we have this case?

Reply:
Ditto first comment i.e. absent a compelling reason one way or the other I would say keep existing behavior. I'm not aware of any documentation for the history of why we have this case. This is another instance where we could check with data but it may not be worth the effort.

Reply:
```
SELECT countif(channel = 'beta')/countif(substr(channel, 0, 4) = 'beta') 
FROM `moz-fx-data-derived-datasets.telemetry.clients_daily_v6` 
where submission_date_s3 = DATE("2019-04-15")
```

Yields 0.9999954487218873

We have achieved 5 nines of reliability that a channel string starting with "beta" will be exactly equal to "beta".

Will leave existing behavior.